dbt/include/fabric/macros/adapters/columns.sql

```diff
@@ -54,8 +54,8 @@
 
 {% macro fabric__alter_column_type(relation, column_name, new_column_type) %}
 
-    {%- set table_name= tmp_relation.include(database=False).include(schema=False)-%}
-    {%- set schema_name = tmp_relation.include(database=False).include(identifier=False) -%}
+    {%- set table_name= relation.include(database=False).include(schema=False)-%}
+    {%- set schema_name = relation.include(database=False).include(identifier=False) -%}
```

Comment:
For simplicity:
```suggestion
    {%- set table_name= relation.identifier -%}
    {%- set schema_name = relation.schema -%}
```
These values won't be quoted, although it looks like you maybe don't want quotes anyway?